### ISSUE TITLE
Is prophet even good for this type of tasks?

### ISSUE THREAD

**Issue by mlinmg** [facebook/prophet#2476]
I'm working on a forecasting problem where my client has data for around 5 years, but the sales events occur sporadically. These sales events typically last about a week out of a month, and both the dates and lengths of the events can vary widely.

I've been trying to use Prophet to forecast future earnings, but I'm encountering difficulties in capturing the right trends. I attempted to use the holidays functionality in Prophet to model these events, but since they can vary significantly, I'm concerned that Prophet might not be able to make accurate forecasts, maybe a better suit would be a DT, what do you think?

**Comment by AlexandroLuis** [comment]
Are you trying to use seasonality? it worked for me to get these gaps of high sales and low, here's an example:
`your_model.add_seasonality(name='weekly',
                                       period=30.5,
                                       fourier_order=6)  # Adjusted Fourier order`

**Comment by mlinmg** [comment]
Yes but this problem does not have seasonality(maybe like weekly but it's not standardised
